web/backend-fastapi/entrypoint.sh

```diff
@@ -3,8 +3,6 @@
 # Load environment variables
 echo "Loading environment variables..."
 
-export AWS_ACCESS_KEY_ID="--"
```

Comment:
As mentioned above. We may need to prune and test this to be certain we don't need this. I believe we had timing issues (fastapi starting without the variables loaded) that we created this file. I'd be cautious removing this before thorough testing. For now, my 2cents is to keep it.  

This is local so.. just an inconvenience more than anything else.

Reply:
Is there a way we can test this? It seems like an accident waiting to happen with someone potentially committing a key.